src/Npgsql/NpgsqlConnector.FrontendMessages.cs

```diff
@@ -132,7 +130,7 @@ internal async Task WriteParse(string sql, string statementName, List<NpgsqlPara
                 if (WriteBuffer.WriteSpaceLeft < 4)
                     await Flush(async);
                 Debug.Assert(p.Handler != null, "Input parameter doesn't have a resolved handler when populating Parse message");
-                WriteBuffer.WriteInt32((int)p.Handler.PostgresType.OID);
+                WriteBuffer.WriteInt32((int)p.Handler!.PostgresType.OID);
```

Comment:
Does the compiler require this? Because we just asserted it the line before...

Reply:
Will remove assert.

Reply:
The assert here does have the message, but I don't really care..